src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarClassification.tsx

```diff
@@ -74,7 +74,7 @@ export const ArtworkSidebarClassificationFragmentContainer = createFragmentConta
     artwork: graphql`
       fragment ArtworkSidebarClassification_artwork on Artwork {
         attribution_class {
-          short_description
+          shortDescription
```

Comment:
Nice, was this a bug?

Reply:
Nah, just a separate deprecation. With our new Apollo tooling this stuff jumps out in VSCode so was able to easily fix.

Reply:
We had a pattern of `_` names which we've deprecated in favor of camel case.